doc/sphinxman/source/glossary_psivariables.rst

```diff
@@ -1290,18 +1308,44 @@ PSI Variables by Alpha
    The total electronic energy [Eh] and correlation energy component [Eh]
    for the orbital-optimized MP2 level of theory.
 
+.. psivar:: OMP2 REFERENCE CORRECTION ENERGY
+
+   The additional correction to the SCF reference energy [Eh]
+   for the orbital-optimized MP2 level of theory.
+
 .. psivar:: OMP2.5 TOTAL ENERGY
    OMP2.5 CORRELATION ENERGY
 
    The total electronic energy [Eh] and correlation energy component [Eh]
    for the orbital-optimized MP2.5 level of theory.
 
+.. psivar:: OMP2.5 REFERENCE CORRECTION ENERGY
+
+   The additional correction to the SCF reference energy [Eh]
+   for the orbital-optimized MP2.5 level of theory.
+
 .. psivar:: OMP3 TOTAL ENERGY
    OMP3 CORRELATION ENERGY
 
    The total electronic energy [Eh] and correlation energy component [Eh]
    for the orbital-optimized MP3 level of theory.
 
+.. psivar:: OMP3 REFERENCE CORRECTION ENERGY
+
+   The additional correction to the SCF reference energy [Eh]
+   for the orbital-optimized MP3 level of theory.
+
+.. psivar:: OREMP2 TOTAL ENERGY
+   OREMP2 CORRELATION ENERGY
+
+   The total electronic energy [Eh] and correlation energy component [Eh]
+   for the orbital-optimized retain the excitation perturbation level of theory.
```

Comment:
On first read, "retain" looks like it functions as a verb instead of part of a name. Possible changes:
1. retain-the-excitation-perturbation
2. REMP (retain the excitation perturbation)
3. Retain the Excitation Perturbation
4. "retain the excitation perturbation"

@behnle, opinions?

This comment applies to **all** REMP variable descriptions.

Reply:
Oh, originally, it was "retaining", so "RE-PT" is the "Retaining the Excitation degree Perturbation Theory" (an alternative and stringent way to derive LCCD as PT, but with the option to go to higher orders, https://doi.org/10.1016/j.cplett.2006.07.081).
REMP is the hybrid between RE-PT and MP-PT.

Reply:
ah, thanks for the clarification. I'm going with `retaining-the-excitation-degree |MollerPlesset| hybrid perturbation theory level`. that, plus `REMP` in the qcvar name should provide enough clues if read carefully to distinguish MPPT, REPT, REMP.